locales/en/yourloops.json

```diff
@@ -79,6 +80,8 @@
   "cancel-remote-monitoring-invite-confirm": "Please, confirm that you want to cancel the monitoring invite for the user {{fullName}}",
   "care-team-membership": "Teams I belong to:",
   "care-team-no-membership": "No care teams",
+  "caregiver-and-family": "Caregiver and family",
+  "caregiver-radio-input": "Caregiver radio selector",
```

Comment:
`Caregiver radio selector` -> `Create caregiver acount`?
This would probably be more explicit to know what the button action is